src/Destructurama.Attributed/Destructurama.Attributed.csproj

```diff
@@ -1,26 +1,29 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard1.1</TargetFramework>
+    <TargetFrameworks>netstandard2.0;netstandard1.1</TargetFrameworks>
+    <LangVersion>latest</LangVersion>
     <AssemblyName>Destructurama.Attributed</AssemblyName>
     <VersionPrefix>2.0.1</VersionPrefix>
```

Comment:
I'll bump this to 3.x post-merge to reflect the potentially-breaking unpinning of the assembly version.

Reply:
What is "unpinning of the assembly version" ?

Reply:
I looked at the previous packages and see that they have an assembly with a different version inside.

Reply:
My mistake - you're quite correct, @sungam3r ; the deletion of https://github.com/destructurama/attributed/blob/master/assets/CommonAssemblyInfo.cs threw me off. I'll switch back from a major to minor revision increment 👍

Reply:
I guessed that it was because of unused https://github.com/destructurama/attributed/blob/master/assets/CommonAssemblyInfo.cs . The same situation is in another package for destructuring. Do you mind if I update it too?

Reply:
That would be great 👍

Reply:
Done: by-ignoring, json-net